api/filter/http/health_check.proto

```diff
@@ -0,0 +1,20 @@
+syntax = "proto3";
+
+package envoy.api.v2.filter.http;
+
+import "google/protobuf/duration.proto";
+
+// Note that the filter will automatically fail health checks and set the
+// x-envoy-immediate-health-check-fail header if the /healthcheck/fail
+// admin endpoint has been called. (The /healthcheck/ok admin endpoint
+// reverses this behavior).
+message HealthCheck {
+  // Specifies whether the filter operates in pass through mode or not.
+  bool pass_through_mode = 1;
```

Comment:
Default?

Reply:
none. This is required.

Reply:
Then it's false, because there's no such thing as required/optional in proto3. Maybe update the docs?